packages/common/src/tests.ts

```diff
@@ -84,3 +89,18 @@ export function getValidInvitationUrlTestData<T extends InvitationData>(data: T)
 //     data: data,
 //   }
 // }
+
+export const createArbitraryFile = (filePath: string, sizeBytes: number) => {
```

Comment:
We can't include this in `common` because `fs` isn't compatible with mobile.  The `Test IOS Build` job is failing because of this.  I had tried to do the same thing and had to split it up in the end.  We may want to create a `backend-common` package at some point.